.github/workflows/macos.yml

```diff
@@ -0,0 +1,30 @@
+name: macOS build
+
+on: [push, pull_request]
+
+jobs:
+  build_gcc9:
+    name: AppleClang [macOS]
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: 'warpx_directory/WarpX'
+    - name: install dependencies
+      run: |
+        brew install cmake
+        brew install fftw
+        brew install libomp
```

Comment:
Do we need these with the `make` command below, or did you just add them so that we don't forget about them in the future?

Reply:
Yep, that was the idea. 
I manually activated OpenMP for now, just to make sure it compiles. I could remove the LDFLAGS and XTRA flags as well as the `libomp` line, which means we have to `USE_OPENMP=FALSE`.

Reply:
Let's keep it as is, that look great, thanks for the explanation!